clients/rest/factory_test.go

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright 2020 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package rest
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/types"
+)
+
+func TestClientFactoryLocal(t *testing.T) {
+	actualClient := ClientFactory(types.EndpointParams{UseRegistry: false}, nil)
```

Comment:
In production usage, I believe ClientFactory's m parameter will always be non-nil.  Test should reflect that.